<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreenWindow</class>
 <widget class="QWidget" name="SplashScreenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>368</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="splashScreenImage">
     <property name="pixmap">
      <pixmap resource="../res/ui.qrc">:/app_splashscreen</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="infoWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="copyrightValue"/>
      </item>
      <item>
       <widget class="QLabel" name="versionValue">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/ui.qrc"/>
 </resources>
 <connections/>
</ui>
